### PR TITLE
fix winedbg buffer-aligned limitation.

### PR DESCRIPTION
**Pull request by XYlearn**

### CODE REVIEW

File: libr/io/p/io_winedbg.c [hunk 1]
```diff
@@ -38,16 +38,22 @@ static int __write(RIO *io, RIODesc *fd, const ut8 *buf, int count) {
...
-		return -1;
-	}
 	for (i = 0; i < words ; i++) {
 		ut64 addr = io->off + (i * wordSize);
 		char *cmd = r_str_newf ("set *0x%"PFMT64x" = 0x%x", addr, w[i]);
 		free (runcmd (cmd));
 		free (cmd);
 	}
+
+    int left = count % wordSize;
```
Comment (radare) [inline libr/io/p/io_winedbg.c]: use tabs not spaces to indent

File: libr/io/p/io_winedbg.c [hunk 2]
```diff
@@ -38,16 +38,22 @@ static int __write(RIO *io, RIODesc *fd, const ut8 *buf, int count) {
...
+    if(left > 0) {
```
Comment (radare) [inline libr/io/p/io_winedbg.c]: add space before every (

File: libr/io/p/io_winedbg.c [hunk 3]
```diff
@@ -38,16 +38,22 @@ static int __write(RIO *io, RIODesc *fd, const ut8 *buf, int count) {
...
+        ut32 leftW = -1;
+        memcpy(&leftW, w + words, left);
```
Comment (radare) [inline libr/io/p/io_winedbg.c]: missing space here

File: libr/io/p/io_winedbg.c [hunk 4]
```diff
@@ -38,16 +38,22 @@ static int __write(RIO *io, RIODesc *fd, const ut8 *buf, int count) {
...
+        ut64 addr = io->off + (words * wordSize);
+        char *cmd = r_str_newf ("set *0x%"PFMT64x" = 0x%x", addr, leftW);
+        free(runcmd (cmd));
```
Comment (radare) [inline libr/io/p/io_winedbg.c]: missing space

**Comment by radare** [comment]
ping

**Comment by radare** [comment]
hello?

**Comment by XVilka** [comment]
@XYlearn ping?

**Comment by XVilka** [comment]
Please rebase instead of merging.

File: libr/io/p/io_winedbg.c [hunk 5]
```diff
@@ -38,16 +38,22 @@ static int __write(RIO *io, RIODesc *fd, const ut8 *buf, int count) {
...
 		free (cmd);
 	}
+
+	int left = count % wordSize;
+	if(left > 0) {
+		ut32 leftW = -1;
+		memcpy(&leftW, w + words, left);
+		ut64 addr = io->off + (words * wordSize);
+		char *cmd = r_str_newf ("set *0x%"PFMT64x" = 0x%x", addr, leftW);
+		free(runcmd (cmd));
```
Comment (XVilka) [inline libr/io/p/io_winedbg.c]: See the difference in code style? Please put a space before `(`

File: libr/io/p/io_winedbg.c [hunk 6]
```diff
@@ -38,16 +38,22 @@ static int __write(RIO *io, RIODesc *fd, const ut8 *buf, int count) {
...
 		ut64 addr = io->off + (i * wordSize);
 		char *cmd = r_str_newf ("set *0x%"PFMT64x" = 0x%x", addr, w[i]);
 		free (runcmd (cmd));
 		free (cmd);
 	}
+
+	int left = count % wordSize;
+	if(left > 0) {
+		ut32 leftW = -1;
+		memcpy(&leftW, w + words, left);
```
Comment (XVilka) [inline libr/io/p/io_winedbg.c]: Please put the space before `(`.

**Comment by radare** [comment]
Why merged . This pr have more issues than just indentation complains

**Pull request merged**